Standalone_GCBM/input_database/add_species_vol_to_bio.py

```diff
@@ -10,53 +11,68 @@
     parser = ArgumentParser(description="Create the deforestation Chile disturbance")
     parser.add_argument("input_db_path", help="GCBM input database path", type=os.path.abspath)
     args = parser.parse_args()
-    
+
     if not os.path.exists(args.input_db_path):
         sys.exit("File not found: {}".format(args.input_db_path))
-    
+
     with sqlite3.connect(args.input_db_path) as conn:
-            
-        
+        cursor = conn.cursor()
         logging.info("Adding a generic tropical species")
         # Create a new species, I will call it "Generic Tropical", ID = 8001, Forest type = 3 (Hardwood) and Genus = 12 (Other broad-leaves species)
-        conn.execute(
-            """
-            INSERT INTO species (id, name, forest_type_id, genus_id)
-            VALUES (8001, 'Generic Tropical', 3, 12)
-            """)
+
+        with open('species.csv','r') as fin1: # `with` statement available in 2.5+
+        # csv.DictReader uses first line in file for column headings by default
+        dr1 = csv.DictReader(fin1) # comma is default delimiter
+        to_db1 = [(i['id'], i['name'], i['forest_type_id'], i['genus_id']) for i in dr1]
+        cursor.executemany(
+            "INSERT INTO species (id, name, forest_type_id, genus_id)
+            VALUES (?, ?, ?, ?)", to_db1
+            )
         # My growth curves have a Red Alter in the species parameter, the id of the red alder is 118, I will replace it with my custom id (8001)
-        conn.execute(
-            """
-            UPDATE growth_curve_component
-            SET species_id = 8001
-            WHERE species_id IN (118)
-            """)
+        with open('growth_curve_component.csv','r') as fin2: # `with` statement available in 2.5+
+        # csv.DictReader uses first line in file for column headings by default
+        dr2 = csv.DictReader(fin2) # comma is default delimiter
+        to_db2 = [(i['species_id'], i['species_id']) for i in dr2]
+
+        cursor.executemany(
+            "UPDATE growth_curve_component
+            SET species_id = ?
+            WHERE species_id = ?", to_db2
+            )

```

Comment:
Here you are replacing the values of the species_id with the same species_id, effectively not changing anything